projects/cashmere/src/lib/button/button.component.scss

```diff
@@ -82,7 +82,7 @@ $btn-icon-sz: 18px;
     &.hc-secondary {
         @include colorButton($gray-100, 5%);
         color: $gray-600;
-        border: 2px solid $gray-300;
+        border: 1.5px solid $gray-300;
```

Comment:
Lets keep our PRs focused on a single component in the future. It made sense to include those other pagination fixes in this but hiding a button change doesn't make sense.